packages/kit/src/core/config/options.js

```diff
@@ -251,7 +251,8 @@ const options = object(
 					`${keypath} has been removed — use the handle hook instead: https://kit.svelte.dev/docs#hooks-handle'`
 			),
 
-			target: string(null),
+			// TODO remove this for 1.0
+			target: error((keypath) => `${keypath} is no longer required`),
```

Comment:
"No longer required" sounds like optional, which doesn't sound like "remove it". This hopefully clarifies it.

```suggestion
			target: error((keypath) => `${keypath} is no longer required. Remove this option.`),
```

Reply:
`${keypath} is no longer supported, You should remove this option` sound better.

Reply:
softened it a bit :)

```suggestion
			target: error((keypath) => `${keypath} is no longer required, and should be removed`),
```